app/assets/javascripts/Components/submission_file_manager.jsx

```diff
@@ -114,22 +117,41 @@ class SubmissionFileManager extends React.Component {
         data.append("grouping_id", this.props.grouping_id);
       }
       data.append("unzip", unzip);
-      $.post({
```

Comment:
It's not clear why you changed this function (or other settings beyond `xhr`)

Reply:
Ah this was unnecessary, stemming from a lack of understanding of jQuery's `ajax`; will revert this but keep the new xhr changes - thanks for pointing out!